docker/docker-python/README.md

```diff
@@ -12,13 +12,21 @@ It uses the appropriate pip binaries to build different docker images as -
 * gpu_cu80_mkl
 * gpu_cu92 
 * gpu_cu92_mkl
+* cpu_py3
```

Comment:
I'd prefer if python3 was the default.

Reply:
I would not want to change the docker images already uploaded for MXNet 1.3 since this might cause something to break for a user. This change to make python3 the default docker version cannot be immediate and should be discussed on the dev list. If agreed upon, this should take effect from the next release onwards. Wdyt?

Reply:
Very good points. Could you start the discussion?

Reply:
Done. The decision is to make python3 default. I will make the change.

Reply:
This has been discussed on dev-list and the decision is to switch to python3 images as default. This will be done in the next release to avoid disruption to the current customers.